### PR TITLE
Azure CI Mac: Disable smoke_LPT/ReduceMinTransformation.CompareWithRe…

### PR DESCRIPTION
**Pull request by azhogov**

### CODE REVIEW

File: .ci/azure/mac.yml [hunk 1]
```diff
@@ -105,7 +105,7 @@ jobs:
     workingDirectory: $(BUILD_DIR)
     displayName: 'Install'
 
-  - script: $(BIN_DIR)/unit-test --gtest_print_time=1 --gtest_filter=-backend_api.config_unsupported:*IE_GPU*:IE_CPU.onnx_model_sigmoid:IE_CPU/GRUSequenceOp.onnx_model_gru*:IE_CPU.exp_* --gtest_output=xml:TEST-NGraphUT.xml
+  - script: $(BIN_DIR)/unit-test --gtest_print_time=1 --gtest_filter=-backend_api.config_unsupported:*IE_GPU*:IE_CPU.onnx_model_sigmoid:IE_CPU/GRUSequenceOp.onnx_model_gru*:IE_CPU.exp_*:smoke_LPT/ReduceMinTransformation.CompareWithRefImpl/f32_Shape* --gtest_output=xml:TEST-NGraphUT.xml
```
Comment (ilya-lavrenov) [inline .ci/azure/mac.yml:108]: looks like it's not a part of unit-tests

Should be disabled here https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

File: .ci/azure/mac.yml [hunk 2]
```diff
@@ -133,7 +133,7 @@ jobs:
     displayName: 'IE FuncTests'
     continueOnError: false
 
-  - script: $(BIN_DIR)/cpuFuncTests --gtest_filter=*smoke* --gtest_print_time=1 --gtest_output=xml:TEST-cpuFuncTests.xml
+  - script: $(BIN_DIR)/cpuFuncTests --gtest_filter=*smoke*:-smoke_LPT/ReduceMinTransformation.CompareWithRefImpl/f32_Shape* --gtest_print_time=1 --gtest_output=xml:TEST-cpuFuncTests.xml
```
Comment (ilya-lavrenov) [inline .ci/azure/mac.yml:136]: to skip test config!

File: .ci/azure/mac.yml (hunk 1 above)
Comment (azhogov) [reply .ci/azure/mac.yml]: fixed

File: .ci/azure/mac.yml (hunk 2 above)
Comment (azhogov) [reply .ci/azure/mac.yml]: moved

**Review by SDxKeeper — APPROVED** [review]

File: inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp [hunk 3]
```diff
@@ -83,6 +83,10 @@ std::vector<std::string> disabledTestPatterns() {
               R"(.*GroupDeconv_3D_Planar_BF16.*K\(1\.1\.1\)_S\(1\.1\.1\).*inFmts=ncdhw_outFmts=ncdhw_primitive=jit_gemm_PluginConf.*)",
               R"(.*GroupDeconv_3D_Planar_BF16.*K\(1\.1\.1\)_S\(2\.2\.2\).*inFmts=ncdhw_outFmts=ncdhw_primitive=jit_gemm_PluginConf.*)",
         });
+#endif
+#ifdef __APPLE__
```
Comment (SDxKeeper) [inline inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp:87]: this code block is not needed anymore

**Review by SDxKeeper — CHANGES_REQUESTED** [review]

File: inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp (hunk 3 above)
Comment (azhogov) [reply inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp:87]: It is needed for investigation of strange fails

**Pull request merged**